### DISCUSSION TITLE
Feature to track how many creatures have been caught (Hunter)

### DISCUSSION THREAD

**Discussion by benxinn** [runelite/runelite#17723] [Ideas]
This can be included in the base Hunter plugin, or in a future Rumor plugin where it could track how many you have caught in the current rumor. There is a similar suggestion about tracking rumours but I think this could be added to the core Hunter plugin too.

**Comment by Birdmanchunk** [comment]
I back this! I've started an eagle's peak one chunk account and tracking hunter npcs looted and the drops would be great!
